cloud/ec2_util.go

```diff
@@ -326,6 +329,12 @@ func cacheHostData(ctx context.Context, h *host.Host, instance *ec2.Instance, cl
 		return errors.Wrap(err, "error updating host document in db")
 	}
 
+	if h.IsVirtualWorkstation {
```

Comment:
why are we doing this just for virtual workstations? I'm not sure we should be limiting this functionality.

Reply:
I didn't know if we wanted to store ipv4 in non virtual workstations but I will put it in for future use! 
I'm still keeping the `SetIPv4Address` separate from `CacheHostData` because I call it when starting instances